sdk/arm/compute-preview/2020-09-30-preview/armcompute/zz_generated_connection.go

```diff
@@ -0,0 +1,12 @@
+// +build go1.13
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+package armcompute
```

Comment:
For the preview modules, should we name it `armcompute` or `armcomputepreview`?

Reply:
Good question, I will follow up about this in scrum.  Also, we had originally planned on previews being a sub-directory of their RP, not a sibling.  i.e. `/sdk/arm/compute/preview/...`.  Of course, this would mean that previews need to be carved out of releases.  I will also discuss having them be siblings.

Reply:
My personal preference is to have the stable and preview modules side by side instead of that the preview module is a sibling of the stable module - this introduces more intuitive rule for the output-folder in readme files also.